src/TwilioVideoParticipantView.ios.js

```diff
@@ -25,7 +25,8 @@ class TwilioVideoParticipantView extends Component {
   }
 
   render () {
-    return <RCTTWRemoteVideoView {...this.props}>{this.props.children}</RCTTWRemoteVideoView>
+    scalesType = this.props.scaleType === 'fit' ? 1 : 2
```

Comment:
Ditto.  This doesn't seem necessary if scaleType is what we're really using.

Reply:
same reasoning